nbcparser/pkg/parser/templates/cse_cmd.sh.gtpl

```diff
@@ -33,7 +33,6 @@ SERVICE_PRINCIPAL_CLIENT_ID={{.AuthConfig.GetServicePrincipalId}}
 NETWORK_PLUGIN={{getStringFromNetworkPluginType .GetNetworkConfig.GetNetworkPlugin}}
 NETWORK_POLICY="{{getStringFromNetworkPolicyType .GetNetworkConfig.GetNetworkPolicy}}"
 VNET_CNI_PLUGINS_URL={{.GetNetworkConfig.GetVnetCniPluginsUrl}}
-CNI_PLUGINS_URL={{.GetNetworkConfig.GetCniPluginsUrl}}
```

Comment:
cc @lilypan26 @anujmaheshwari1 for bootstrap contract changes

Reply:
hoping rp can drop this too.